autoload/rekkr.wad/bloodcolor.deh

```diff
@@ -0,0 +1,5 @@
+Thing 15 (Cacodemon)
```

Comment:
How about the Hell Knight?

Reply:
Hell Knight from Doom 2, Rekkr is Doom 1.

Reply:
Alright. I have read somewhere that it uses some states from Doom 2 things, but it can't spawn them on maps. Thanks!

Reply:
I'm not sure myself. Maybe this needs more testing.